packages/algob/src/lib/tx.ts

```diff
@@ -100,7 +100,12 @@ function signTransaction(txn: Transaction, signer: wtypes.Sign): Uint8Array {
 			return txn.signTxn(signer.fromAccount.sk);
 		}
 		case wtypes.SignType.LogicSignature: {
-			signer.lsig.lsig.args = signer.args ?? [];
+			if (signer?.lsig?.lsig?.args) {
+				signer.lsig.lsig.args = signer.args ?? [];
+			} else {
+				(signer.lsig as any).args = signer.args ?? []; // args property didn't exist in earlier version of API (for reference: see the multisig example)
```

Comment:
same here

Reply:
Actually it is necessary since the new type of LogicSigAccount type consist of args, due to which it was giving me a type error which casting it to any